pre_commit/languages/r.py

```diff
@@ -88,13 +88,13 @@ def install_environment(
     env_dir = _get_env_dir(prefix, version)
     with clean_path_on_failure(env_dir):
         os.makedirs(env_dir, exist_ok=True)
-        path_desc_source = prefix.path('DESCRIPTION')
-        if os.path.exists(path_desc_source):
-            shutil.copy(path_desc_source, env_dir)
         shutil.copy(prefix.path('renv.lock'), env_dir)
         cmd_output_b(
             'Rscript', '--vanilla', '-e',
-            """\
+            f"""\
+            prefix_dir <- "{
+                prefix.prefix_dir.encode('unicode_escape').decode()
+            }"
```

Comment:
I think  you can use `{prefix.prefix_dir!r}` to do this as well -- that'll give you the quotes and then you won't have to do an encoding dance

Reply:
Nice. I really start to like Python string manipulations 😀. All checks pass now.